engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetViewportToSizeOf.java

```diff
@@ -96,6 +96,12 @@ private FBO getFbo() {
         return fboManager.get(fboName);
     }
 
+    public static void ResetDefaultInstance() {
```

Comment:
I'm really sorry for the capitalization, idk what I was thinking.
Must be my left-over habit from the pre-Terasology era :P